src/transforms/html-min-transform.js

```diff
@@ -14,7 +14,7 @@ https://github.com/fluid-project/trivet/raw/main/LICENSE.md.
 const htmlmin = require("html-minifier");
 
 module.exports = function htmlMinTransform(value, outputPath) {
-    if (outputPath.indexOf(".html") > -1) {
+    if (outputPath && outputPath.includes(".html")) {
```

Comment:
Probably doesn't matter but might be more appropriate/safer to use [`endsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith). That way you just check the end of the string, instead of seeing if `.html` appears anywhere in the `outputPath`.